packages/web_drivers/test/chrome_driver_installer_test.dart

```diff
@@ -33,4 +33,13 @@ void main() async {
 
     expectLater(command.isInstalled, isTrue);
   });
+
+  test('get chrome version', () async {
```

Comment:
How does this test work? It is marked as `@TestOn('vm && linux')` above 🤔 

Also, where do we run these tests? I don't see any tests in the list of checks for the PR.

Reply:
@TestOn is not effective if you run it yourself manually. I did TDD. I was using this method when I was developing.

There is no CI for this repository, I can remove the test if you want to.

I personally believe it might be useful for future contributors though if they are also doing TDD.

Reply:
No, we should definitely run tests. We should just mark this as `skip: !Platform.isMacOS` or similar.

@godofredoc Can we add this repo to LUCI?

Reply:
done (added the skip)